### DISCUSSION TITLE
https://alpinejs.dev/components is throwing a 500 error

### DISCUSSION THREAD

**Discussion by darylknight** [alpinejs/alpine#3698] [7. Bugs (archived)]
Sorry I'm not sure how else to reach you. I'm working on a site where I need to reference the tabs component, and the Components paid area is offline with a 500 error:

https://alpinejs.dev/components

![image](https://github.com/alpinejs/alpine/assets/1250017/f8007b34-c39b-47f1-ac6b-c76657118b64)

**Comment by sphkress** [comment]
any updates regarding this issue?

**Comment by darylknight** [comment]
It's working for me again
